core/shared/src/main/scala/magnolia.scala

```diff
@@ -136,6 +139,10 @@ object Magnolia {
 
     val fullauto = c.macroApplication.symbol.isImplicit
 
+    // halp! I really just want to ask "is t a subtype of T?"
+    def halfauto(s: Type): Boolean =
```

Comment:
I don't think this works in the general case. Can't you use the `<:<` method?

Reply:
the "halp" was definitely directed at you :smile: I'll try that...